### PR TITLE
Console: setHealthCommandHandler

### PR DESCRIPTION
**Pull request by Mpcs**
# Description
 New command - `setHealth value`. Sets hero's life to the specified value. Has to be between 0 and max Health

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/java/org/destinationsol/game/console/commands/SetHealthCommandHandler.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if (newHealthValue < 0f) {
+            console.warn("Cannot set health to less than zero");
+            return;
+        }
+        if (hero.isDead()) {
+            console.warn("Cannot set health when hero is dead");
+            return;
+        }
+
+        hero.getShip().setLife(newHealthValue);
```
Comment (Avalancs) [inline engine/src/main/java/org/destinationsol/game/console/commands/SetHealthCommandHandler.java]: Does this work when the hero is in transcendent mode?

Comment (Avalancs) [reply engine/src/main/java/org/destinationsol/game/console/commands/SetHealthCommandHandler.java]: Just tested, it throws an exception. please add a check for transcendence

**Comment by NicholasBatesNZ** [comment]
If you have some time, would you mind making a new PR for this command? Your new console work is in `develop` now :)

**Pull request closed without merging**